intra/dialers/weird_split.go

```diff
@@ -0,0 +1,88 @@
+package dialers
+
+import (
+	"io"
+	"net"
+	"strings"
+	"syscall"
+)
+
+type WeirdSplitter struct {
+	*net.TCPConn
+	Used bool
+	Size int
```

Comment:
A code comment on how clients should set this field? To half of MSS/MTU? Or, some arbitrary value greater than 2 will do?

Reply:
I don't know. `splitHello()` choose where the upstream segment to be split randomly.
https://github.com/celzero/firestack/blob/n2/intra/dialers/retrier.go#L343-L359
And should I rename this field position?